SECURITY.md

```diff
@@ -0,0 +1,70 @@
+# Security Policy
+
+
+## What is a vulnerability?
+
+All vulnerabilities are bugs, but not every bug is a vulnerability. Vulnerabilities compromise one or more of:
+
+- Confidentiality (personal or corporate confidential data).
+- Integrity (trustworthiness and correctness).
+- Availability (uptime and service).
+
+If you discover a security vulnerability within Multipass, we encourage responsible disclosure. 
+If you're not sure whether you found a vulnerability, a bug, or neither, please use the process below for reporting a vulnerability. 
+We will then assess and triage accordingly.
+
+## Reporting a Vulnerability
+
+Multipass accepts private reports of security vulnerabilities made through 
+[GitHub Security Advisories](https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/about-repository-security-advisories). 
+Here is the link to open a [new security adivory](https://github.com/canonical/multipass/security/advisories/new). 
+For detailed instructions, please review the documentation on [privately reporting a security vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability).
+
+### Steps to Report a Vulnerability
+
+1. Go to the [Security Advisories Page](https://github.com/canonical/multipass/security/advisories) of the `multipass` repository.
+2. Click "Report a Vulnerability."
+3. Provide detailed information about the vulnerability, including steps to reproduce, affected versions, and potential impact.
+
+## Response to vulnerabilities
+
+Vulnerabilities are classified by risk, which factors in impact and likelihood. 
+The Multipass project will prioritize responding to all High and Critical severity vulnerabilities.
+
+When we receive an issue, we will work with the reporter to determine how best to proceed. 
+After a fix is available to a confirmed vulnerability, we will also coordinate disclosure with releasing to the various platforms.
```

Comment:
This sentence is a bit confusing to me. Should the `with` be an `and`?

Reply:
The intention was that the processes of disclosing and releasing had to be coordinated with each other. But I suppose it's not achieving that anyway, so I'll replace as you suggest.